node/examples/electron/image_classification/main.js

```diff
@@ -18,14 +18,22 @@ function createWindow() {
     }
   })
 
-  // Load the index.html with 'numRuns' to run inference multiple times.
+
   let url = `file://${__dirname}/image_classification/index.html`
+  let numRunsParam, deviceParam
   for (let argv of process.argv) {
-    if (argv.startsWith("numRuns")) {
-      url += '?' + argv
-      break
+    if (argv.startsWith("numRuns=") && !numRunsParam) {
+      // Load the index.html with 'numRuns' to run inference multiple times.
+      numRunsParam = argv
```

Comment:
These codes may look confused. How about using a list/array to store these params, then generating the url by it?

Reply:
And I think it's no necessary to check whether users set duplicated params.

Reply:
Thanks @mingmingtasd, done in latest commit. PTAL.